### PR TITLE
Keep CrashpadClient around for application lifetime

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

**Comment by Atlante45** [comment]
## Test Plan:
- Verify simulated crashes still get to backtrace.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9192/app_13007_PR_65fe65d0cca419ca864e8ecd35e95d63823d6d2e.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9192/HighFidelity-Beta-PR13007-65fe65d0cca419ca864e8ecd35e95d63823d6d2e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9192/HighFidelity-Beta-Interface-PR13007-65fe65d0cca419ca864e8ecd35e95d63823d6d2e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9192/HighFidelity-Beta-PR13007-65fe65d0cca419ca864e8ecd35e95d63823d6d2e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9192/HighFidelity-Beta-Interface-PR13007-65fe65d0cca419ca864e8ecd35e95d63823d6d2e.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Fail @conklin94122 
12 crashes were executed sequentially and only 8 were returned.
In addition, some crashes might have been labeled incorrectly. Attached is 2 logs that state different crashes than what was performed in the dev >crash menu.
[Backtrace - 8 or 12 captures.pdf](https://github.com/highfidelity/hifi/files/1961568/Backtrace.-.8.or.12.captures.pdf)
[26d6.txt](https://github.com/highfidelity/hifi/files/1961570/26d6.txt)
[26d0.txt](https://github.com/highfidelity/hifi/files/1961571/26d0.txt)

File: interface/src/Crashpad.cpp [hunk 1]
```diff
@@ -11,42 +11,42 @@
...
 
-static std::wstring gIPCPipe;
-
 extern QString qAppFileName();
 
 // crashpad::AnnotationList* crashpadAnnotations { nullptr };
 
-#include <Windows.h>
+
+std::unique_ptr<CrashpadClient> client;
```
Comment (kencooke) [inline interface/src/Crashpad.cpp]: I don't think we want this to be wrapped in a unique_ptr.
If you look at the implementation, the pointer dereference is declared ```noexcept``` and we are calling it from an exception handler. From the MSFT docs:
```
If an exception does reach a function marked noexcept, std::terminate is invoked immediately and there is no guarantee that destructors of any in-scope objects will be invoked.
```

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9328/HighFidelity-Beta-PR13007-250806252e526889820549e6ac13ce259a340ebd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9328/HighFidelity-Beta-Interface-PR13007-250806252e526889820549e6ac13ce259a340ebd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9328/HighFidelity-Beta-PR13007-250806252e526889820549e6ac13ce259a340ebd.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9339/app_13007_PR_f3fe1f3e58386363ddb2e2b5a48e5a08bd73f4c3.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9339/HighFidelity-Beta-PR13007-f3fe1f3e58386363ddb2e2b5a48e5a08bd73f4c3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9339/HighFidelity-Beta-Interface-PR13007-f3fe1f3e58386363ddb2e2b5a48e5a08bd73f4c3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9339/HighFidelity-Beta-PR13007-f3fe1f3e58386363ddb2e2b5a48e5a08bd73f4c3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9339/HighFidelity-Beta-Interface-PR13007-f3fe1f3e58386363ddb2e2b5a48e5a08bd73f4c3.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by Atlante45** [comment]
@qlimnavarro @conklin94122 Given the nature of some of those crashes, we won't be able to catch 100% of them.
What we do get from that report is satisfying, this is a PASS.

**Pull request merged**